### ISSUE TITLE
Suspicious "$href" in example collection_info_example.json

### ISSUE THREAD

**Issue by rouault** [opengeospatial/ogc_api_coverages#91]
https://github.com/opengeospatial/ogc_api_coverages/search?q=%22%24href%22 shows that "$href" constructs are found in the following pages:
https://github.com/opengeospatial/ogc_api_coverages/blob/788d7adeb71b09166077cb9937548e5ee3152174/standard/examples/JSON/collection_info_example.json

I assume this should  be "href" instead (without leading dollar), or perhaps I'm missing something

I'm also seeing "$href" in schemas. But perhaps this is expected there, although I can't find this mentioned in https://swagger.io/specification/ (just "$ref") ?
https://github.com/opengeospatial/ogc_api_coverages/blob/a3ee10d52aa0bf95b383ad4e1b79ba2e73823079/standard/openapi/schemas/common/collections.json
https://github.com/opengeospatial/ogc_api_coverages/blob/a3ee10d52aa0bf95b383ad4e1b79ba2e73823079/standard/openapi/schemas/common/collectionInfo.json
https://github.com/opengeospatial/ogc_api_coverages/blob/788d7adeb71b09166077cb9937548e5ee3152174/standard/openapi/schemas/coverage_collectionInfo.json
https://github.com/opengeospatial/ogc_api_coverages/blob/bbb77dd6f2ed929dc13a97b708ca9c7309a52456/standard/openapi/schemas/common/landingPage.json

**Comment by jerstlouis** [comment]
@rouault @Schpidi
As per resolution of #64, the decision was to use the usual OGC API links mechanism to link to external properties, e.g.:
```json
    "links" : [
        {
            "type": "application/json",
            "rel": "http://www.opengis.net/def/rel/ogc/1.0/coverage-domainset",
            "title": "Coverage domain set of collection in CIS JSON",
            "href" : "http://data.example.org/collections/CIS123456789/coverage/domainset"
        }
    ]
```
and when the link is to a local property in the doucment instead to use a URL like:
```json
            "rel": "http://www.opengis.net/def/rel/ogc/1.0/coverage-domainset",
            "href" : "#domainset"
```

So those domainset / rangetype "$href" should be "href" and be moved to the elements of the links property array.

I believe those other `$href` are all typos which should be `$ref` (where content is replaced, as in OpenAPI schemas).

Searching for "$href" indeed only returns OGC (typos?) results :)

**Comment by jerstlouis** [comment]
Fixed in https://github.com/opengeospatial/ogc_api_coverages/pull/98

**Comment by Schpidi** [comment]
Coverages SWG call: Should be fixed with #98.
